src/Components/OurTeams/LeaderDescription.css

```diff
@@ -0,0 +1,21 @@
+* {
+    font-family: -apple-system, BlinkMacSystemFont, "Roboto", sans-serif;
+}
+.leaderDesc {
+    font-weight: 700;
+    line-height: 38px;
+}
+.leaderName {
+    font-size: 38px;
+    color: #2e244b;
+}
+.leaderPosition {
+    font-size:21px;
```

Comment:
font size should change with screen size. Refer home page to see how font changes with screen size